storage/google/cloud/storage/bucket.py

```diff
@@ -578,11 +578,14 @@ def id(self):
 
     @property
     def lifecycle_rules(self):
-        """Lifecycle rules configured for this bucket.
+        """Retrieve or set lifecycle rules configured for this bucket.
 
         See https://cloud.google.com/storage/docs/lifecycle and
              https://cloud.google.com/storage/docs/json_api/v1/buckets
 
+        :setter: Set lifestyle rules for this bucket.
+        :getter: Gets the lifestyle rules for this bucket.
```

Comment:
:+1: I didn't realize this was part of Sphinx. Win.

@tseaver Have you checked that this docstring (vs. the one from the setter) "wins"?